contracts/core/L2Core.sol

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.4;
+
+import {Vcon} from "../vcon/Vcon.sol";
+import {IVolt, Volt, IERC20} from "../volt/Volt.sol";
+import {ICore} from "./ICore.sol";
+import {Permissions} from "./Permissions.sol";
+
+/// @title Source of truth for VOLT Protocol on Arbitrum
```

Comment:
If this comment is correct, this contract should be named something like ArbitrumCore, not L2Core